### PR TITLE
fix doc generation; generate docs

### PR DESCRIPTION
**Pull request by TylerGillson**

### CODE REVIEW

**Comment by TylerGillson** [comment]
Looks like no one has run `go generate` in a very long time 😢

**Review by karl-cardenas-coding — APPROVED** [review]
THe majority of the changes are indentation and whitespace cleanup. Other than that, the most notable change is the removal of cluster_nested.md and the addition of virtual_cluster.md


Really big fan of this change. Thank you so much @TylerGillson

File: examples/e2e/eks-static/terraform.template.tfvars [hunk 1]
```diff
@@ -66,8 +66,3 @@ worker_azs_subnets_map = {
   "{enter AWS Availability Zone A}" = "{enter Subnet for AZ A, ...}"
   "{enter AWS Availability Zone B}" = "{enter Subnet for AZ A, ...}"
 }
-
- master_azs_subnets_map = {
```
Comment (nikchern) [inline examples/e2e/eks-static/terraform.template.tfvars:70]: why this is deleted?

**Review by nikchern — CHANGES_REQUESTED** [review]

File: examples/e2e/eks-static/terraform.template.tfvars (hunk 1 above)
Comment (TylerGillson) [reply examples/e2e/eks-static/terraform.template.tfvars:70]: It was causing `go generate` to fail because that key is already present in the same file.

**Review by nikchern — APPROVED** [review]

**Pull request merged**